exaudfclient/base/javacontainer/BUILD

```diff
@@ -52,6 +62,7 @@ genrule(
             TMPDIR=`mktemp -d`
             cp -r -L $(location //:filter_swig_code.py) $(locations :exascript_java_tmp_cc) $$TMPDIR
             (cd $$TMPDIR &&
+            tar xf java_src.tar
             python3 filter_swig_code.py "exascript_java.cc" "exascript_java_tmp.cc")
             cp $$TMPDIR/exascript_java.cc "$(location exascript_java.cc)"
             cp -r $$TMPDIR/java_src/com/exasol/swig/ConnectionInformationWrapper.java "$(location ConnectionInformationWrapper.java)"
```

Comment:
extract cp of java code to single artifacts from filter rule if still needed, because it is unrelated to the filtering

Reply:
created a separate rule `finish_java_package` which copies now all Java files together